src/cls/IPM/Main.cls

```diff
@@ -2951,22 +2953,36 @@ ClassMethod DisplayModules(ByRef pList, pNumbered As %Boolean = 0, pWithNamespac
     $ListBuild("Repository", $$$Blue)
   )
 
+  Kill serverVersions
+  If $Data(pModifiers("showupstream"), repoName) && ##class(%IPM.Repo.Definition).ServerDefinitionKeyExists(repoName) {
```

Comment:
Can we show latest available across all configured repos? Or with a default value if there's no value?